coroutines-codelab/start/src/main/java/com/example/android/kotlincoroutines/main/TitleRepository.kt

```diff
@@ -51,7 +51,7 @@ class TitleRepository(val network: MainNetwork, val titleDao: TitleDao) {
      * the current tile.
      */
     fun refreshTitleWithCallbacks(titleRefreshCallback: TitleRefreshCallback) {
```

Comment:
https://codelabs.developers.google.com/codelabs/kotlin-coroutines/index.html?index=..%2F..index#8

In the codelab section 9 said,
> Delete the `refreshTitleCallbacks` function as it is not used anymore.

But the real function name is `refreshTitleWithCallbacks`. 
In my opinion, `refreshTitleWithCallbacks` seems much better name, because it exactly explains what it does.
How about replacing `refreshTitleCallbacks` by `refreshTitleWithCallbacks` in the codelab?

Reply:
Will update text, thanks!